### DISCUSSION TITLE
Standard template little concern

### DISCUSSION THREAD

**Discussion by ScriBanana** [argenos/zotero-mdnotes#81] [Q&A]
Hi,

Very nice to see the update, great work! The template and placeholders customisation features are amazing, an excellent idea.
I also greatly appreciate to see our concerns be addressed. Thank you.
The default update stuck to 0.1.0 so I had the ghost link issue until updating to 0.1.2 but that's a small concern.

A very little issue : the standard template is written is such a fashion that if the zotero file is not tagged (uncommon, I know) the resulting "Tag" line on the MD will be `, #zotero, #literature-notes, #reference` instead of `- Tags: #zotero, #literature-notes, #reference` for example.

By the way, I think that `{{abstractNote}}` is missing from the placeholders dictionary of the wiki.

Edit : Removed my previous comments on `{{noteContent}}` to make a proper ticket.

**Comment by argenos** [comment]
`{{abstractNote}}` is a default field from Zotero, so it's not included in the placeholders added by mdnotes.

As for the tags, I would suggest customizing the template. Perhaps it makes sense to separate the tags of the Zotero item from those added to the markdown file. You could also customize the placeholder content to have the manually added tags before the tags of the Zotero item.

**Comment by ScriBanana** [comment]
Alright thanks. I do customised the template and I reckon most of us tag our Zotero entries (and don't use the template standard tags anyway), it just is a small inconsistency in the template.
Modifying the placeholder is a good workaround.
